tests/test_corner.py

```diff
@@ -157,6 +157,20 @@ def test_truths():
     _run_corner(truths=[0.0, None, 0.15])
 
 
+@image_comparison(
+    baseline_images=["reverse_truths"], remove_text=True, extensions=["png"]
+)
+def test_reverse_truths():
+    _run_corner(truths=[0.0, None, 0.15], reverse=True)
+
+
+@image_comparison(
+    baseline_images=["truths"], remove_text=True, extensions=["png"]
+)
+def test_truths():
```

Comment:
It looks like this test is a duplicate of a previous one?

Reply:
My bad -- it's been duplicated; I am really not entirely sure how -- just delete it

Reply:
:+1: Will do!